test/functional/api/command_spec.lua

```diff
@@ -51,39 +49,39 @@ describe('nvim_get_commands', function()
   before_each(clear)
 
   it('gets empty list if no commands were defined', function()
-    eq({}, meths.get_commands({ builtin = false }))
+    eq({}, api.nvim_get_commands({ builtin = false }))
```

Comment:
At some point I would like to make it clearer which things are going to the nvim test-instance (the "[SUT](https://en.wikipedia.org/wiki/System_under_test)"). Then it's much easier for contributors to understand which calls are going to the SUT and which are executed in the test-runner.

E.g. something like:

```
sut.nvim_get_commands instead of helpers.api.nvim_get_commands
sut.fn.foo() instead of helpers.fn.foo()
```

And we should rename `helpers.lua` to `test.lua` and change the import convention to:

```
local t = helpers

...
t.foo()
t.bar()
````